### DISCUSSION TITLE
Is there a way to make search use "Search all" by default?

### DISCUSSION THREAD

**Discussion by ForceConstant** [searxng/searxng#1978] [Q&A] [answered]
An annoying thing that modern search engines do, is I search for 2 words, and it returns things that only have one of the words in it, and doesn't even state that, so then I have to go back and add a '+' to the less common word. Is there a setting in SearXNG so I can make the default to require all the words?

**Comment by return42** [comment, marked as answer]
Sorry no / keep in mind; SearXNG is a meta search engine and search terms are simply pass through the underlying search engines.
